cmake/Modules/FindUDev.cmake

```diff
@@ -29,9 +29,9 @@ IF (UDEV_LIBRARIES AND UDEV_INCLUDE_DIR)
     execute_process(COMMAND pkg-config --atleast-version=143 libudev RESULT_VARIABLE UDEV_STABLE)
     # retvale is 0 of the condition is "true" so we need to negate the value...
     if (UDEV_STABLE)
-	set(UDEV_STABLE 0)
+    set(UDEV_STABLE 0)
     else (UDEV_STABLE)
-	set(UDEV_STABLE 1)
+    set(UDEV_STABLE 1)
```

Comment:
The script itself is distributed under "the BSD license" whichever version it is. Modifications of this form aren't problematic as long as we keep it in the public repository.

As for the indenting, it was only "good" if you had the right tab spacing, otherwise it would end up looking inconsistent with the first level of indenting already. I'll fix the spacing.